VulnerableApplication/Controllers/HomeController.cs

```diff
@@ -30,5 +30,8 @@ public ActionResult Contact()
         public string Hello() {
         return "this is not on the coding rules.";
         }
+        
+        private static string ChangePassword = "PassWord917231";
```

Comment:
[VULNERABILITY] 'password' detected in this expression, review this potentially hardcoded credential.

Reply:
/workitem

Reply:
WorkItem 767 Created see [workItem](https://dev.azure.com/csedevops/ea38b5b8-2b4f-4721-8514-fe765aa4a23a/_apis/wit/workItems/767)